components/camel-huawei/camel-huaweicloud-obs/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add missing LICENSE header

Reply:
Added missing license header in pom file